src/styles/index.scss

```diff
@@ -1,3 +1,123 @@
-body {
+.page {
+  --main-accent-color: #00acdc;
+  --blue-accent-color: #060b35;
+  --secondary-color: #616070;
+
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-color);
+}
+
+.page__body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
```

Comment:
Ensure that the SCSS file is being compiled to a CSS file and that the correct CSS file is being linked in the HTML. The HTML file currently links to `index.scss`, which should be a compiled CSS file instead.